CMakeLists.txt

```
@@ -22,28 +50,17 @@ target_compile_definitions(${LIBRARY} PUBLIC
   -DUSE_HAL_DRIVER
   -DSTM32H723xx
   -DDEBUG
+  ${ETH}
 )
 
-option(BOARD "Board clock config" OFF)
-option(NUCLEO "Nucleo clock config" OFF)
 
-if(BOARD)
-  add_definitions(-DBOARD)
-  add_definitions(-DHSE_VALUE=25000000)
-endif()
-
-if(NUCLEO)
-  add_definitions(-DNUCLEO)
-  add_definitions(-DHSE_VALUE=8000000)
-endif()
 
 
 target_compile_options(${LIBRARY} PUBLIC
   -mcpu=cortex-m7
   -mfpu=fpv5-d16
   -mfloat-abi=hard
   -mthumb
-  
   -Og
```

If this is hardcoded, there is no use in setting the option above. Change also -g3
```suggestion
  -${OPTIMIZATION}
```

---

fixed